dnf-boot.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+"$1"/usr/bin/clr-boot-manager update -p "$1"
```

Comment:
Inclined to call the file dnf-post-cbm.sh

Reply:
Maybe pkg-post-update.sh?  Since it would also be running the update helper and cbm update?  pkg since in theory, we could add support for additional package managers?

Reply:
Ah yes sure